src/test/java/org/prebid/server/auction/AmpRequestFactoryTest.java

```diff
@@ -361,6 +343,654 @@ public void shouldRespondWithBidRequestWithTestFlagOn() {
         assertThat(captor.getValue().getTest()).isEqualTo(1);
     }
 
+    @Test
+    public void shouldReturnBidRequestWithOverriddenTagIdBySlotParamValue() {
+        // given
+        given(httpRequest.getParam("slot")).willReturn("Overridden-tagId");
+        given(storedRequestProcessor.processAmpRequest(anyString()))
+                .willReturn(Future.succeededFuture(givenBidRequestWithExt(null,null)));
+        given(auctionRequestFactory.fillImplicitParameters(any(), eq(routingContext)))
```

Comment:
In general there is no necessity to specify concrete instance in mock setup (`eq(routingContext)`) unless mock behavior should differ depending on the instance passed to it.